### ISSUE TITLE
[feature request] Allow for a difference of one in alignment with concat_equals, plus_equals etc.

### ISSUE THREAD

**Issue by Dacaspex** [squizlabs/PHP_CodeSniffer#1941]
Currently, PHPCS gives a warning for the following code sample
```
$foo = 'foo';
$foo .= 'bar';
```
```
Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space
```
In our company, we allow for this code style. I would like to request a feature to add an option that we can disable this warning for this specific case. 

This is the case for concat equals (.=), plus equals (+=), minus equals (-=), division equals (/=), times equals (*=), etc.

### Example samples ###
```
$foo = 'foo';
$foo .= 'bar';

$foo .= 'foo';
$bar = 'bar';

$foo = 1;
$foo += 2;
$foo += 5;
$foo -= 2;
$bar = 6;
```

**Comment by gsherwood** [comment]
The alignment in your examples would drive me crazy :)

This isn't a feature I would add personally, but I'd consider merging a PR if there is a lot of support for it. Otherwise, maybe the sniff could be refactored to allow you to write a custom sniff that extends it but requires a different number of spaces in these cases.

**Comment by Dacaspex** [comment]
Thanks you for considering. I wrote a hacky solution for the time being. I understand that this isn't exactly what you want, but let's see what other people think of it

**Comment by gsherwood** [comment]
A new sniff property will be in 3.6.0 that makes this possible. It is documented here: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#genericformattingmultiplestatementalignment
